vms/avm/network/gossip.go

```diff
@@ -23,6 +23,10 @@ var (
 	_ gossip.Marshaller[*txs.Tx] = (*txParser)(nil)
 )
 
+// bloomChurnMultiplier is the number used to multiply the size of the mempool
+// to determine how large of a bloom filter to create.
+const bloomChurnMultiplier = 3
```

Comment:
I see this is defined same in other vm/mempools. Should we move it to bloom package as a exported constant?

Reply:
I hadn't really considered this a "standard" value. I think this depends on the workload and we shouldn't codify it in `bloom`.